src/Nethermind/Nethermind.GitBook/RpcAndCliGenerator.cs

```diff
@@ -0,0 +1,293 @@
+                        _markdownGenerator.CreateTab(rpcBuilder, $"Example response of {rpcMethodName}");
+                        _markdownGenerator.CreateCodeBlock(rpcBuilder, exampleResponse);
+                        _markdownGenerator.CloseTab(rpcBuilder);
+                    }
+                    
+                    if (typesToDescribe.Count != 0)
+                    {
+                        objectsBuilder.AppendLine();
+                        _markdownGenerator.CreateTab(objectsBuilder, $"Objects in {rpcMethodName}");
+                        _sharedContent.AddObjectsDescription(objectsBuilder, typesToDescribe);
+                        _markdownGenerator.CloseTab(objectsBuilder);
+                    }
+
+                    rpcBuilder.Append(objectsBuilder);
+                    _markdownGenerator.CloseTabs(rpcBuilder);
+                    rpcBuilder.AppendLine();
+                    
+                    if (methodData.InvocationType == InvocationType.Both)
+                    {
+                        rpcBuilder.AppendLine($"[See also CLI {cliMethodName}](https://docs.nethermind.io/nethermind/nethermind-utilities/cli/{moduleName}#{moduleName.ToLower()}-{methodName.ToLower()})");
+                        
+                        CreateCliContent(cliBuilder, paramBuilder, returnBuilder, objectsBuilder, cliMethodName, exampleArguments, cliInvocation, methodData);
+                        cliBuilder.AppendLine($"[See also JSON RPC {rpcMethodName}](https://docs.nethermind.io/nethermind/ethereum-client/json-rpc/{moduleName}#{rpcMethodName.ToLower()})");
+                    }
+                }
+                else
+                {
+                    Type returnType = methodData.ReturnType;
+                    string returnTypeToWrite = _sharedContent.GetTypeToWrite(returnType, typesToDescribe);
+                    returnBuilder.AppendLine("| Returned type | Description |");
+                    returnBuilder.AppendLine("| :--- | :--- |");
+                    returnBuilder.AppendLine(@$"| `{returnTypeToWrite}` | {methodData.ResponseDescription} |");
+
+                    CreateCliContent(cliBuilder, paramBuilder, returnBuilder, objectsBuilder, cliMethodName, exampleArguments, cliInvocation, methodData);
+                }
+            }
+
+            const int notEmptyModule = 20;
+
+            if (rpcBuilder.Length > notEmptyModule)
+            {
+                _sharedContent.Save(moduleName, string.Concat(docsDir, "/ethereum-client/json-rpc"), rpcBuilder);
+            }
+            if (cliBuilder.Length > notEmptyModule)
+            {
+                _sharedContent.Save(moduleName, string.Concat(docsDir, "/nethermind-utilities/cli"), cliBuilder);
+            }
+        }
+
+        private string CreateRpcExampleResponse(string methodDataExampleResponse)
+        {
+            string exampleResponseForRpc = methodDataExampleResponse.Replace("\n", "\n  ");
```

Comment:
Environment.NewLine here instead of \n

Reply:
Here I'm converting CLI example to RPC example and this replace operates on string with "\n" inside and just adding two more spaces on the beginning of every line to have RPC example formatted well after adding

{
  "jsonrpc":  "2.0",
  "result":

on the beginning, not sure if changing it have sense